eureka/src/main/java/com/linecorp/armeria/server/eureka/InstanceInfoBuilder.java

```diff
@@ -241,9 +251,10 @@ InstanceInfo build() {
         final LeaseInfo leaseInfo = new LeaseInfo(renewalIntervalSeconds, leaseDurationSeconds);
         return new InstanceInfo(instanceId, appName, appGroupName, hostname, ipAddr, vipAddress,
```

Comment:
so the constructor needs to be accessible outside the package because interestingly the builder for instanceinfo is not in the same package as instanceinfo! Another way is to move this builder to common, then either re-use it here as-is, or extend or delegate to the common one.

Another way is to make a special copy factory function in instanceinfo like.. `InstanceInfo.WithHealthCheckUrlPath`

depends on what you feel is best (or something else, @minwoox!)

Reply:
Oops, I missed it. 😓 Let's leave it as is.
I will make the constructor package-private after https://github.com/line/armeria/pull/4243 is merged.